Cannot Compile llm-base on main

Trying to run the base example on macOS and cannot use main version package a v0.1.1. 

```bash
 Compiling llm-base v0.2.0-dev (https://github.com/rustformers/llm?rev=39eb341aeda6a3ff0240421e54df2707ae8743fc#39eb341a)
error[E0308]: mismatched types
   --> /Users/ccaarrll/.cargo/git/checkouts/llm-d8a8bbe144aa0546/39eb341/crates/llm-base/src/tokenizer/huggingface.rs:25:21
    |
25  |             .decode(vec![idx as u32], true)
    |              ------ ^^^^^^^^^^^^^^^^ expected `&[u32]`, found `Vec<u32>`
    |              |
    |              arguments to this method are incorrect
    |
    = note: expected reference `&[u32]`
                  found struct `Vec<u32>`
note: method defined here
   --> /Users/ccaarrll/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokenizers-0.13.4/src/tokenizer/mod.rs:814:12
    |
814 |     pub fn decode(&self, ids: &[u32], skip_special_tokens: bool) -> Result<String> {
    |            ^^^^^^

error[E0308]: mismatched types
   --> /Users/ccaarrll/.cargo/git/checkouts/llm-d8a8bbe144aa0546/39eb341/crates/llm-base/src/tokenizer/huggingface.rs:70:21
    |
70  |             .decode(tokens, skip_special_tokens)
    |              ------ ^^^^^^ expected `&[u32]`, found `Vec<u32>`
    |              |
    |              arguments to this method are incorrect
    |
    = note: expected reference `&[u32]`
                  found struct `Vec<u32>`
note: method defined here
   --> /Users/ccaarrll/.cargo/registry/src/index.crates.io-6f17d22bba15001f/tokenizers-0.13.4/src/tokenizer/mod.rs:814:12
    |
814 |     pub fn decode(&self, ids: &[u32], skip_special_tokens: bool) -> Result<String> {
    |            ^^^^^^
help: consider borrowing here
    |
70  |             .decode(&tokens, skip_special_tokens)
    |                     +

For more information about this error, try `rustc --explain E0308`.
error: could not compile `llm-base` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

--- a/crates/llm-base/Cargo.toml
+++ b/crates/llm-base/Cargo.toml
@@ -22,7 +22,7 @@ partial_sort = "0.2.0"
 serde_bytes = "0.11"
 memmap2 = { workspace = true }
 half = "2"
-tokenizers = {version="0.13.3", default-features=false, features=["onig"]}
+tokenizers = {version="0.13.4", default-features=false, features=["onig"]}
 regex = "1.8"
 tracing = { workspace = true }
 

--- a/crates/llm-base/src/tokenizer/huggingface.rs
+++ b/crates/llm-base/src/tokenizer/huggingface.rs
@@ -22,7 +22,7 @@ impl HuggingFaceTokenizer {
     /// Converts a token index to the token it represents in this tokenizer.
     pub(crate) fn token(&self, idx: usize) -> Vec<u8> {
         self.tokenizer
-            .decode(vec![idx as u32], true)
+            .decode(&[idx as u32], true)
             .expect("Cannot decode token from tokenizer tokenizer.")
             .as_bytes()
             .to_vec()
@@ -67,7 +67,7 @@ impl HuggingFaceTokenizer {
     /// Decode a list `tokens` with this tokenizer.
     pub(crate) fn decode(&self, tokens: Vec<TokenId>, skip_special_tokens: bool) -> Vec<u8> {
         self.tokenizer
-            .decode(tokens, skip_special_tokens)
+            .decode(&tokens, skip_special_tokens)
             .expect("Cannot decode token from tokenizer.")
             .as_bytes()
             .to_vec()

